modules/restore-cert-roxctl.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/restore-acs.adoc
+:_content-type: PROCEDURE
+[id="restore-cert-roxctl_{context}"]
+= Restore certificates using the `roxctl` CLI
+
+[role="_abstract"]
+Use the `roxctl` CLI to generate Kubernetes manifests to install the {product-title-short} Central component to your cluster.
+Doing this allows you to ensure that authentication certificates for Secured clusters and the API tokens remain valid for the restored version.
+If you backed up another instance of {product-title-short} Central, you can use the certificate files from that backup.
+
+[NOTE]
+====
+With the `roxctl` CLI, you can not restore the entire Central deployment. Instead, first you use the `roxctl` CLI to generate new manifests using the certificates in your central data backup. Afterwards, you use those manifests to install Central.
+====
+
+.Prerequisites
+* You must have the `central-tls` backup file.
```

Comment:
```suggestion
* You must have a Red Hat Advanced Cluster Security for Kubernetes backup file.
```
The certificates are also store in the RHACS backup file. When using roxctl, it loads the certificates from this backup file (the database backup). So you don't need the central-tls manifest backup file.